mock-esignet-integration-impl/src/main/java/io/mosip/esignet/mock/integration/service/MockKeyBindingWrapperService.java

```diff
@@ -109,6 +172,34 @@ public KeyBindingResult doKeyBinding(String individualId, List<AuthChallenge> ch
         return keyBindingResult;
     }
 
+    private boolean isSupportedOtpChannel(String channel) {
+        return ("email".equalsIgnoreCase(channel) || "mobile".equalsIgnoreCase(channel));
+    }
+
+    public boolean authenticate(IdentityData identityData, List<AuthChallenge> challengeList) throws KeyBindingException {
```

Comment:
instead of duplicating this method in mockAuthenticator and mockKeyBinder, move this method to mockHelperService.

Reply:
moved isSupportedOtpChannel method into mockHelperService.
authenticate is duplicated from mock-identity system, not getting referred in any other service.